src/Symfony/Bundle/FrameworkBundle/Tests/Functional/ApiAttributesTest.php

```diff
@@ -365,7 +674,17 @@ public function __invoke(#[MapQueryString] ?QueryString $query): Response
     }
 }
 
-class WithMapRequestPayloadController
+class WithMapQueryStringWithDefaultValueController
+{
+    public function __invoke(#[MapQueryString] QueryString $query = new QueryString(new Filter('approved', 5))): Response
```

Comment:
To ensure clarity, it’s not the default value that makes this work, but the argument type itself (`QueryString $query`). I think your PR title/description is not yet clear about this. In fact, having a empty query string + default value will skip the validation.